app/views/supervisor/subjects/_subject_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_for @subject, url: supervisor_subjects_path do |f| %>
+  <%= render "shared/error_messages", object: f.object %>
+
+  <%= f.label :name, Subject.human_attribute_name("name") %>
+  <%= f.text_field :name, class: "form-control", required: true %>
+
+  <%= f.label :description, Subject.human_attribute_name("description") %>
+  <%= f.text_area :description, class: "form-control", required: true, cols: 40, rows: 8 %>
+
+  <div class="row">
+    <div class="col-md-4">
+      <%= f.label :question_amount, Subject.human_attribute_name("question_amount") %>
+      <%= f.number_field :question_amount, class: "form-control", required: true %>
+    </div>
+    <div class="col-md-4">
+      <%= f.label :pass_score, Subject.human_attribute_name("pass_score") + " (%)" %>
+      <%= f.number_field :pass_score, class: "form-control", required: true %>
```

Comment:
chỗ này em set thêm min vào để cho ng dùng khỏi nhập số âm dc em 
[number_field](https://stackoverflow.com/questions/28134186/ruby-on-rails-how-to-set-size-of-number-field)